cirq-google/cirq_google/ops/fsim_via_model_tag_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cirq
+import cirq_google
+
+
+def test_equality():
+    assert cirq_google.FSimViaModelTag() == cirq_google.FSimViaModelTag()
+    assert hash(cirq_google.FSimViaModelTag()) == hash(cirq_google.FSimViaModelTag())
+
+
+def test_syc_str_repr():
```

Comment:
Rename to `test_syc_str_repr()` since syc is not related.

Reply:
Yeah, sorry for the copy-paste error.